src/components/TagPicker.vue

```diff
@@ -56,5 +58,18 @@ export default class TagPicker extends Vue {
   get layers() {
     return this.store.layers;
   }
+
+  $refs!: {
+    combobox: any;
```

Comment:
Using `HTMLFormElement` for the type would be preferable

Reply:
Thanks!